### PR TITLE
Update media bias labels

### PR DESCRIPTION
**Pull request by meddulla**
This is an initial proposal for the labels' remapping. Most of the remapping is done around 88983242f327d4d27ae2c46af56c3d3807b16b0e and the commits before.

I didn't see a mapping of "FAILED FACT CHECKS" but maybe it's no longer present in media bias. If it's still there I may have missed something...

I also did some code cleanup, added logging and added a requirements file among others. Some of this was to help me better understand the code and may come down to personal preferences. Happy to change it again so it's clearer :)

And the relevant issue: https://github.com/Data4Democracy/are-you-fake-news/issues/15

### CODE REVIEW

**Comment by meddulla** [comment]
Forgot to say but did this in the `develop_dockerize` branch. Think I saw you say somewhere that's where new development should be done but can't find where right now. Hopefully it's all good anyway :)

File: docker/gather_data/join_source_lists.py [hunk 1]
```diff
@@ -2,12 +2,96 @@
...
+        ('very low', 'low'),
+        ('poor sourcing', 'low'),
+        ('extreme bias', 'low'),
+        ('propaganda', 'low'),
+
+        # High
+        ('high (no pun intended)', 'high'),
+        ('high', 'high'),
+        ('very high', 'high'),
+        ('very-high', 'high'),
```
Comment (N2ITN) [inline docker/gather_data/join_source_lists.py]:
```suggestion
        ('very-high', 'very high'),
```

File: docker/gather_data/join_source_lists.py [hunk 2]
```diff
@@ -2,12 +2,96 @@
...
+        ('low', 'low'),
+        ('very low', 'low'),
+        ('poor sourcing', 'low'),
+        ('extreme bias', 'low'),
+        ('propaganda', 'low'),
+
+        # High
+        ('high (no pun intended)', 'high'),
+        ('high', 'high'),
+        ('very high', 'high'),
```
Comment (N2ITN) [inline docker/gather_data/join_source_lists.py]:
```suggestion
      
```
Suggest removing this line; as "very high" is a legitimate category.

**Comment by meddulla** [comment]
@N2ITN Updated the pr. Detected that some pages weren't being processed due to their tags not being in the `entry-content` class, so changed it to using the `entry` class instead. Also, added mappings for the new categories, including `Failed fact checks` - some I was unsure so happy to discuss if need be. Left the `re-evaluated-sources` as unmapped.

**Comment by N2ITN** [comment]
Thanks for your hard work!

**Pull request merged**